app/models/profile_declaration.rb

```diff
@@ -4,11 +4,21 @@ class ProfileDeclaration < ApplicationRecord
   delegated_type :declarable, types: %w[EarlyCareerTeacherProfile MentorProfile]
   belongs_to :participant_declaration
 
-  scope :uplift, lambda {
-                   joins(
-                     "left join early_career_teacher_profiles
-                      on declarable_type='EarlyCareerTeacherProfile'
-                      and declarable_id=early_career_teacher_profiles.id",
-                   ).merge(EarlyCareerTeacherProfile.uplift)
-                 }
+  scope :ect_profiles, lambda {
+    joins(
+      "left join early_career_teacher_profiles
+       on declarable_type='EarlyCareerTeacherProfile'
+       and declarable_id=early_career_teacher_profiles.id",
+    )
+  }
+
+  scope :mentor_profiles, lambda {
+    joins(
+      "left join mentor_profiles
+       on declarable_type='MentorProfile'
+       and declarable_id=mentor_profiles.id",
+    )
+  }
```

Comment:
Can we point them at new classes before Stan ends up deleting the old classes? Maybe in a separate PR, I just don't want it to get lost.

Reply:
Is that his final naming? Having them inherit from the eigenclass, I thought, was just an interim position. I'll check with @BroiSatse.